src/sentry/api/endpoints/organization_details.py

```diff
@@ -698,6 +701,9 @@ def put(self, request: Request, organization) -> Response:
                 access=request.access,
             )
 
+            if not include_feature_flags:
+                context.pop("features", None)
```

Comment:
Is there a way to prevent getting the data in the first place? This helps reduce the amount of bytes transferred on the network/wire and reduced the payload, but we are still taking a hit on the data lookup. Is that something you plan to do on a follow up?

Reply:
yeah, i am planning to modify the OrganizationSerializer so that I pass the qparam there and generate the list if the param is true.

Reply:
Okay, awesome! Is that going to be in this PR or another? Just so I know how to review

Reply:
ill follow up with another pr.